base/vulkanexamplebase.h

```diff
@@ -50,9 +50,59 @@
 #define GAMEPAD_BUTTON_R1 0x1005
 #define GAMEPAD_BUTTON_START 0x1006
 
+#if defined(__linux__) && !defined(__ANDROID__)
+
+#define KEY_W 25
+#define KEY_A 38
+#define KEY_S 39
+#define KEY_D 40
+
+#define KEY_Q 24
+#define KEY_E 26
+#define KEY_F 41
+#define KEY_B 56
+#define KEY_O 32
+#define KEY_N 57
+#define KEY_T 28
+#define KEY_L 46
+
+#define KEY_LEFT 113
+#define KEY_RIGHT 114
+#define KEY_UP 111
+#define KEY_DOWN 116
+
+#define KEY_SPACE 65
+#define KEY_RETURN 36
+#define KEY_ESC 9
+
+#define KEY_LSHIFT 50
+#define KEY_RSHIFT 62
+
+#define KEY_NP_PLUS 86
+#define KEY_NP_MINUS 82
+
+#else
```

Comment:
I think it would be better to explicitly say what those codes are for.
If I understand correctly, the `#else` branch here if for Windows, so `#elif defined(_WIN32)` would be better IMO.